pypeit/specobj.py

```diff
@@ -372,18 +373,56 @@ def med_s2n(self):
                 break
         return SN
 
-    def med_fwhm(self):
+    def med_fwhm(self, platescale):
```

Comment:
I think the `SpecObj` object should know about the platescale.  Can we make this an attribute or a property that uses the spectrograph to get it?

Reply:
I added it as a property.